spec/services/alma/indexer_spec.rb

```diff
@@ -56,4 +56,23 @@
       expect(response['response']['numFound']).to eq 1
     end
   end
+
+  describe "incremental_index!" do
+    it "indexes a dump's files" do
+      # url = Rails.application.config.solr['url']
```

Comment:
@hackartisan I reverted back to the old solr_url syntax (rather than using the value from Rails's config). If this works I can add a separate issue to figure out why the Rails config version does not work in Circle CI.

Reply:
👍🏻 thanks!